### PR TITLE
Add keras freeze graph model for tensorflow example

### PR DESCRIPTION
**Pull request by adamcavendish**
Some issues have mentioned that they have some need to freeze graph model to use for tensorflow. I'm providing an example here. 

https://github.com/fchollet/keras/issues/4491

### CODE REVIEW

**Comment by fchollet** [comment]
Does not clearly belong in core Keras; please consider submitting to Keras contrib repo instead.

**Comment by adamcavendish** [comment]
Where is the keras contrib?

File: examples/freeze-graph.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    tfckptcb = TFCheckpointCallback(tf_saver, sess)
+
+    # Write the protobuf graph
+    # ref: https://www.tensorflow.org/api_docs/python/tf/train/write_graph
+    tf.train.write_graph(tf_graph.as_graph_def(),
+                         'freeze', 'graph.pbtxt', as_text=True)
+    tf.train.write_graph(tf_graph.as_graph_def(),
+                         'freeze', 'graph.pb', as_text=False)
+
+    # Train it!
```
Comment (fulkast) [inline examples/freeze-graph.py]: Does this mean that the exported graph doesn't have the post-training weights?

Comment (fulkast) [inline examples/freeze-graph.py]: Just realized that a freezing step is required to freeze the graph_def with the trained weights.

**Pull request closed without merging**